src/app/zones/views/ZoneEgressListView.vue

```diff
@@ -13,128 +15,94 @@
         },
       ]"
     >
-      <div class="zoneegresses">
-        <div class="stack">
-          <div class="kcard-border">
-            <DataOverview
-              :selected-entity-name="entity?.name"
-              :page-size="PAGE_SIZE_DEFAULT"
-              :is-loading="isLoading"
+      <DataSource
+        v-slot="{ data: zoneEgressOverviewData, error }: ZoneEgressOverviewCollectionSource"
+        :src="`/zones/zone-egresses?size=${props.size}&page=${props.page}`"
+      >
+        <KCard>
+          <template #body>
+            <AppCollection
+              data-testid="zone-egress-table"
+              :headers="HEADERS"
```

Comment:
Can we just put the headers in here please, it makes it so much easier to see what is going on.

Reply:
I changed this, but I don’t understand what this brings you. But I don’t argue this again.